pkg/util/labels.go

```diff
@@ -0,0 +1,16 @@
+package util
```

Comment:
pkg/util/labels.go what maybe not the good choice.

What about a new pkg/utils tree where we would move all xxxutil packages, ie.:

- pkg/utils/labelutil/*.go
- pkg/utils/runutil/*.go
- pkg/utils/strutil/*.go
...

What do you think @bwplotka ?

Reply:
just flattening this to 

* pkg/labelutil/*.go
* pkg/runutil/*.go
* pkg/strutil/*.go

sounds good to me as well.